admin-dev/filemanager/include/php_image_magician.php

```diff
@@ -2497,7 +2497,7 @@ public function reset()
 
 ## --------------------------------------------------------
```

Comment:
Some `#` left, I thought this was fixed in another PR that hase been merged https://github.com/PrestaShop/PrestaShop/pull/11699
problem during rebase maybe?